librz/type/parser/c_cpp_parser.c

```diff
@@ -52,6 +52,7 @@ void c_parser_state_free(CParserState *state) {
 }
 
 void c_parser_state_free_keep_ht(CParserState *state) {
+	ht_pp_free(state->forward);
```

Comment:
I don't think it's a good idea since we might need forward-looking definitions between various SDB imports or `td` commands. Better to keep this between parser runs

Reply:
you should check your code.
that value gets every time overwritten by a new pointer.

Reply:
Ok, then it's fine.